lib/sms_autofill.dart

```diff
@@ -184,6 +186,8 @@ class _PhoneFieldHintState extends State<PhoneFieldHint> {
   bool _isUsingInternalController = false;
   bool _isUsingInternalFocusNode = false;
 
+  InputDecoration _decoration;
```

Comment:
no need to keep a reference in the state, should be directly used in the build method. Any particular reason to keep it here ?

Reply:
Good shout.

Reply:
@jaumard - sorry for slow reply. I took a look at your comment above again this morning and remembered the reason; if decoration is not provided it defaults to the existing styling to not break existing usages in other projects. Checkout line 204.

Reply:
but for that you don't need to keep the reference, it can be done directly on the build method :) 
inputDecoration is not a controller so no need to keep reference in the state.

Reply:
Think this is now how you wanted it.

Reply:
yes thanks!